package.json

```diff
@@ -112,12 +112,12 @@
     "eslint-plugin-flowtype": "^2.35.1",
     "eslint-plugin-import": "^2.7.0",
     "eslint-plugin-prettier": "^2.2.0",
-    "eslint-plugin-react": "^7.3.0",
+    "eslint-plugin-react": "^7.4.0",
     "express": "^4.15.4",
     "fake-indexeddb": "^2.0.3",
     "file-loader": "^0.11.2",
-    "flow-bin": "^0.52.0",
-    "flow-coverage-report": "^0.3.0",
+    "flow-bin": "^0.54.1",
+    "flow-coverage-report": "^0.4.0",
```

Comment:
The upgrade makes an error in `flow-coverage-report` disappear. Alternatively we could ignore this module. But newer flow versions (not v0.54) actually ignore modules that aren't actual dependencies to the code so I chose this path.